### DISCUSSION TITLE
How extract the pre-populated database file from local Storage?

### DISCUSSION THREAD

**Discussion by sagarbanik** [simolus3/drift#2177] [Q&A]
As the documentation:

 if (!await file.exists()) {
        // Extract the pre-populated database file from assets
        final blob = await rootBundle.load('assets/my_database.db');
        final buffer = blob.buffer;
        await file.writeAsBytes(buffer.asUint8List(blob.offsetInBytes, blob.lengthInBytes));
}

I'm now okay with this where my database is in the app asset directory. But I want to achieve this for the local storage database file.

My SQLite DB directory is as follows: 
/storage/emulated/0/Download/db.sqlite . I want to import this file instead of asset/db.sqlite file.

Advance thanks for your reply.

**Comment by simolus3** [comment]
I'm not sure I understand what you want to do? You have a file at `/storage/emulated/0/Download/db.sqlite` that you want to use as an initial database? `await File('/storage/emulated/0/Download/db.sqlite').copy(file.path)`?
